### PR TITLE
Bump unzipper and nodejs versions

### PR DESCRIPTION
**Pull request by austinvanderpool**
[Lokalise pulls have been failing for a few days now with a "Premature close" error](https://github.com/color/color/actions/runs/9845697846). After a bit of digging, I found [this issue in the unzipper package repo](https://github.com/ZJONSSON/node-unzipper/issues/269) describing the same error that we’re getting. We’re [supposedly running this action](https://github.com/color/lokalise-actions/blob/main/.tool-versions) on node 16 so I didn’t think that issue would apply. However, I saw a warning on the CI failure linking to [this changelog](https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/) stating that the action would start running on node 20 by default.

In this PR I'm bumping us to the most recent version of both nodejs and unzipper. This fixes tests locally, and hopefully allows us to pull from lokalise again (without breaking pushes 😁).

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1 +1 @@
-nodejs 16.4.2
+nodejs 20.11.0
```
[the remark this answers is not in the document]
Comment (austinvanderpool) [reply .tool-versions]: good call

File: .tool-versions [hunk 2]
```diff
@@ -1 +1 @@
-nodejs 16.4.2
+nodejs 20.15.1
```
[the remark this answers is not in the document]
Comment (austinvanderpool) [reply .tool-versions:1]: haha yes it is, I'm not a fan of such a major jump at once. I went ahead and bumped it due to this warning I saw on the pull action:
image

(which links here https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/)

**Review by DavidMikeSimon — APPROVED** [review]

File: .tool-versions (hunk 2 above)
[the remark this answers is not in the document]
Comment (austinvanderpool) [reply .tool-versions:1]: so part of me wonders if it's been running on node 20 anyways

**Review by skluck-color — COMMENTED** [review]
You'll also need to bump the node version in https://github.com/color/lokalise-actions/blob/main/action.yml#L36 since .tool-versions only has impact on this repo's CI, but not when the action is actually used.

**Pull request merged**